src/github.com/stellar/horizon/db/asset.go

```diff
@@ -7,8 +7,8 @@ import (
 
 // This file contains helpers for working with stellar assets
 
-func assetFromDB(typ int32, code string, issuer string) (result xdr.Asset, err error) {
-	switch xdr.AssetType(typ) {
+func assetFromDB(typ xdr.AssetType, code string, issuer string) (result xdr.Asset, err error) {
+	switch typ {
```

Comment:
`type` is a reserved word in go.  I've been using `typ` and `aType` randomly to work around it.